frontend/src/scenes/data-management/ingestion-warnings/IngestionWarningsView.tsx

```diff
@@ -17,14 +17,16 @@ export const scene: SceneExport = {
 
 const WARNING_TYPE_TO_DESCRIPTION = {
     cannot_merge_already_identified: 'Refused to merge an already identified user via $identify or $create_alias call',
+    cannot_merge_with_illegal_distinct_id:
+        'Refused to merge with an illegal distinct_id via $identify or $create_alias call',
+    cannot_merge_attepts_exhausted: 'Merge via $identify or $create_alias call failed',
+    skipping_event_invalid_uuid: 'Refused to process event with invalid uuid',
 }
 
 const WARNING_TYPE_RENDERER = {
     cannot_merge_already_identified: function Render(warning: IngestionWarning): JSX.Element {
         const details = warning.details as {
-            sourcePerson: string
```

Comment:
This is a bit tricky. 
If we added the person.uuid at the warning time, then we run into the problem that the uuid might not later match to that distinctID - person can later be deleted (via a merge) or split (via split ids).
So the way we can do this is having those lookups in the API before we send the data over to the UI, for which we'd have similar functions for each error type or perhaps try to lookup all properties that include 'distinctID' and add the corresponding 'personUuid' field to details. Since this currently doesn't work that way I suggest we change it later when relevant (unless you know it's planned in the immediate future)

Reply:
>If we added the person.uuid at the warning time, then we run into the problem that the uuid might not later match to that distinctID - person can later be deleted (via a merge) or split (via split ids).

That's true... Hmm. While this is hard to fix after the fact (since existing data would be junk) the reasoning here makes sense - let's continue as-is.